File: image-sizer-impl/src/main/java/com/steelzack/image/sizer/ImageSizerOptions.java
package com.steelzack.image.sizer;

import org.kohsuke.args4j.Option;

/**
 * 
 * @author JOAO
 *
 */
public class ImageSizerOptions {

	@Option(name = "-s", aliases = "--source-file", required = true, help = true)
	private String sourceFile;

	@Option(name = "-d", aliases = "--destination-file", required = true, help = true)
	private String destinationFile;

	@Option(name = "-dim", aliases = "--dimensions", usage = "<width>x<height>")
	private String dimensions;

	public String getSourceFile() {
		return sourceFile;
	}

	public String getDestinationFile() {
		return destinationFile;
	}

	public String getDimensions() {
		return dimensions;
	}
}

File: jeorg-kotlin-apps/image-sizer/backup/2022011image-sizer-impl-java-awt/src/main/kotlin/org/jesperancinha/images/sizer/ImageSizerOptions.kt
package org.jesperancinha.images.sizer

import org.kohsuke.args4j.Option

/**
 *
 * @author JOAO
 */
class ImageSizerOptions {
    @Option(name = "-s", aliases = ["--source-file"], required = true, help = true)
    val sourceFile: String? = null

    @Option(name = "-d", aliases = ["--destination-file"], required = true, help = true)
    val destinationFile: String? = null

    @Option(name = "-dim", aliases = ["--dimensions"], usage = "<width>x<height>")
    val dimensions: String? = null
}